XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
+This chapter treats the technologies ‘payment channels’, ‘atomic swaps’ the basic functionality of the decentralized order book. XUD exposes a simple API designed for exchanges and, together with XUD nodes of other exchanges, forms the decentralized network connecting the exchanges. XUD is available as [installable software](https://github.com/ExchangeUnion/xud/wiki/Installation) and will additionally be provided in form of a docker container which packages all necessary things to operate XUD.
 
 
 3.1. Payment Channels
 ---------------------
-The idea of payment channels is around for a while, even Satoshi Nakamoto already explored [*high-frequency transactions*](https://en.bitcoin.it/wiki/Payment_channels), then with bidirectional channels and the inception of the **Lightning Network** by Joseph Poon and Tadge Dryja, payment channels became more widely known. In short, payment channels enable instant, off-chain transfer of ownership of a digital token, like bitcoin, directly between two parties. Payment channels are opened and closed via one on-chain transactions each. Once a bidirectional payment channel is open, it uses unbroadcasted, fully valid signed transactions, which are sent back and forth between the two parties. These are held on each side of the channel, in our case on the exchange’s XUD, representing the balance of each party. These transactions will use a multi-signature address as their input (the funding address) and they will point at two different addresses for their output, each controlled by one party of the payment channel.
+The idea of payment channels is around for a while, even Satoshi already explored [*high-frequency transactions*](https://en.bitcoin.it/wiki/Payment_channels). Then, with bidirectional channels, the inception of today's **Lightning Network** by Joseph Poon and Tadge Dryja, payment channels became more widely known. In short, payment channels enable instant, off-chain transfer of a digital token or coin, like bitcoin, directly between *two* parties. Payment channels are opened and closed via one on-chain transaction. A payment channel uses unbroadcast, but signed and fully valid on-chain transactions, which are sent back and forth between the two parties to do transactions. Since these unbroadcast transactions are merely data packets which are send via the internet layer and don't need to be validated on a blockchain, transfers are as fast as the internet connection between the two parties.
```

Comment:
"has been around" instead of "is around", and don't need "already" in "already explored".

Second sentence can use active voice and fewer commas like "In 2016, Joseph Poon and Tadge Dryja introduced the concept for the Lightning Network using long-lived, bidirectional payment channels."